content/docs/manuals/networking-internals/mqtt-authentication.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "MQTT Authentication"
+summary: ""
+draft: false
+weight: 315
+toc: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+## Intra-Cluster Communication
+
+![MQTT Architecture Picture](MQTT-Arch.png)
+
+The above image shows how intra-cluster communication takes place. Nodes send application status and node health reports to the cluster-service-manager.
+The cluster-service-manager then appropriately propagates this information to other nodes and the root-service-manager. Since the nodes report sensitive information, such
+as IP Addresses, we highly recommend securing the MQTT channels.
+
+{{< callout context="note" title="MQTT" icon="outline/info-circle">}} 
+MQTT is a lightweight messaging Protocol that supports publishing/subscribing to named channels. Oakestra uses MQTT due to it's minimal network usage and low processing overhead. More information at [mqtt.org](https://mqtt.org/).
+{{< /callout >}}
+
+
+## MQTTS
+MQTT supports the exchanging of certificates to establish a TLS secured channel. For this the server (MQTT Broker) and every 
+client require a certificate-keyfile pair singed against the same certificate Authority (CA).
+The MQTT broker can be configured to only accept incoming secured connection, and to identify devices by their certificate common name (CN) entry.
+
+## Enable Mosquitto Authentication
+
+{{< callout context="caution" title="Requirements" icon="outline/alert-triangle">}} 
+* You have a running Oakestra deployment
+* You have at least one worker node registered
+* (Optional) The NetManager is installed and properly configured
+{{< /callout >}}
+
+{{< link-card
+  title="Getting Started Guide"
+  description="Check out the Getting Started guide to set up your first cluster"
+  href="../../../getting-started/oak-environment/create-your-first-oakestra-orchestrator"
+  target="_blank"
+>}}
+
+
+### Configuring the Cluster Manager
+
+
+Navigate into the `cluster_orchestrator` directory in the oakestra repository.
+
+1. Configure the MQTT Broker by adding the following lines to the `mosquitto/mosquitto.conf` file:
+    ```bash
```

Comment:
For this concrete case I recommend to use `yml` instead of bash.

This is not really bash nor yaml.
But conf has no nice indentation.

bash has a strange coloring to it here

![image](https://github.com/user-attachments/assets/cd9487f2-e3b3-43c7-b4d7-d023bf66eb69)

where as yml still provides the easy copy button and is homogeneous
![image](https://github.com/user-attachments/assets/071d4f1f-1766-42e4-9da4-c746b6214bea)